docs/spec-conformance.md

```diff
@@ -1,13 +1,15 @@
 # Spec conformance
 
-This branch of runc implements the [OCI Runtime Spec v1.2.1](https://github.com/opencontainers/runtime-spec/tree/v1.2.1)
+This branch of runc implements the [OCI Runtime Spec v1.3.0](https://github.com/opencontainers/runtime-spec/tree/v1.3.0)
 for the `linux` platform.
 
 The following features are not implemented yet:
 
 Spec version | Feature                                        | PR
 -------------|------------------------------------------------|----------------------------------------------------------
 v1.1.0       | `SECCOMP_FILTER_FLAG_WAIT_KILLABLE_RECV`       | [#3862](https://github.com/opencontainers/runc/pull/3862)
+v1.3.0       | Clarified interpretation of `linux.intelRdt`   | [#3832](https://github.com/opencontainers/runc/pull/3832)
+v1.3.0       | Fail on a failure of a poststart hook.         | [#4348](https://github.com/opencontainers/runc/pull/4348)
 
```

Comment:
We're also still missing the pids limit 0 support stuff, I will try to send something soon...

Reply:
I think we will need to punt it for 1.4.1, but the issue is that we cannot update the `specs-go` package to v1.3.0 when it is released because the change to `*int64` causes build failures.

Reply:
We have to release runc v1.4.0 GA with the spec v1.3.0 GA, so anyway we have to fix the build failures at least